### DISCUSSION TITLE
Add line breaks / newline for cell entries being of type array

### DISCUSSION THREAD

**Discussion by kalaschnik** [TanStack/table#5287] [General] [answered]
Some of my cell data may either be `string | string[]` data, or likewise `number | number[]`. So far, the data is being rendered (react-style) as: A,B,C or 1,2,3 in the cell.

Is there to render the data as such:

A
B
C

likewise:

1
2
3

I added a code sandbox link of fresh next/shadcn setup:
https://codesandbox.io/p/devbox/vigilant-lumiere-9kvwvw?file=%2Fapp%2Fpage.tsx%3A9%2C19

<img width="755" alt="image" src="https://github.com/TanStack/table/assets/9831678/75d3624a-efad-40e2-bdc5-76756758e870">

In that example, there can be multiple email fields and I want to show them line-by-line. How can I intercept that in tanstack table v8?

In other words, I need a `<br>` between the elements of the array, so that the outputs looks like this:
<img width="638" alt="image" src="https://github.com/TanStack/table/assets/9831678/a3e06800-c2b7-48f4-a34b-3adb825b5248">

**Comment by kalaschnik** [comment, marked as answer]
Alright, after digging a bit, I realized I can modify the cell property:

```diff
{
    accessorKey: "email",
    header: "Email",
},
{
    accessorKey: "email",
    header: "Email",
+    cell: ({ getValue }) => (
+      <span>
+        {Array.isArray(getValue()) ? (
+          <ul>
+            {(getValue() as string[]).map((item: string, index: number) => (
+              <li key={index}>• {item}</li>
+            ))}
+          </ul>
+        ) : (
+          (getValue() as React.ReactNode)
+        )}
+      </span>
+    ),
  },
```

The changes are in the link from top.
